packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,410 @@
+<template>
+    <div
+        ref="cookieBanner"
+        :class="[$style['c-cookieBanner'], $style['c-cookieBanner-overlay'], { [$style['c-cookieBanner--hidden']]: hideBanner }]"
+        attr="data-cookie-consent-overlay"
+        :aria-hidden="isHidden">
+        <div
+            v-if="!legacyBanner"
+            :class="$style['c-cookieBanner-card']">
+            <div
+                id="dialog1"
```

Comment:
We probably need a more descriptive `id` here.

Reply:
the `id` is a hangover from the GTM version and can be removed